google-cloud-spanner-executor/pom.xml

```diff
@@ -82,6 +90,14 @@
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-cloud-spanner-executor-v1</artifactId>
       <version>1.4.0</version>
+      <!--   Temp exclusion. Remove next protobuf upgrade (to 3.23.3 or higher)
```

Comment:
1. How would we ensure that this temp exclusion gets removed? Do we have a good internal tracking mechanism?
2. How do we know that this may not introduce any other side effect? Any side effect that can be caused by excluding `j2objc-annotations` dependency? Or is this transitively getting included?

Reply:
* Sure thanks for the clarification and having this transitively included ensures there are no others side effects. 
* Yes, internal ticket works to track removing this workaround.